src/read.c

```diff
@@ -3184,8 +3591,10 @@ static avifResult avifParse(avifDecoder * decoder)
     avifBool ftypSeen = AVIF_FALSE;
     avifBool metaSeen = AVIF_FALSE;
     avifBool moovSeen = AVIF_FALSE;
+    avifBool coniSeen = AVIF_FALSE;
     avifBool needsMeta = AVIF_FALSE;
     avifBool needsMoov = AVIF_FALSE;
+    avifBool needsConi = AVIF_FALSE;
```

Comment:
Can we put `coniSeen` and `needsConi` inside `#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)`?